packages/frontend-2/components/projects/AddDialog.vue

```diff
@@ -114,24 +125,21 @@ const emit = defineEmits<{
 
 const isWorkspacesEnabled = useIsWorkspacesEnabled()
 const createProject = useCreateProject()
-const { handleSubmit } = useForm<FormValues>()
+const router = useRouter()
+const { handleSubmit, meta } = useForm<FormValues>()
 const { result: workspaceResult } = useQuery(projectWorkspaceSelectQuery, null, () => ({
   enabled: isWorkspacesEnabled.value
 }))
 
 const visibility = ref(ProjectVisibility.Unlisted)
 const selectedWorkspace = ref<ProjectsAddDialog_WorkspaceFragment>()
-const isCreatingWorkspace = ref<boolean>(false)
+const showConfirmDialog = ref(false)
```

Comment:
@andrewwallacespeckle What is the difference between these two? Can it be the same modal, with just a different action on `confirm`?

Reply:
I did it this way because we don't know what the user is confirming when there is 1 confirm modal. To me it made more sense having 2 separate instances of the confirm modal, as they are confirming different things (1 to close dialog, 1 to navigate to workspace explainer). I've updated this to use 1 modal, but had to introduce a new ref to track what they are confirming.